docs/Actions.md

```diff
@@ -53,7 +53,7 @@ await comment.markAsSpam()
 
 ## Batch updates
 
-Whenever you make more than one change (create or update records) in an action, you should **batch them**.
+Whenever you make more than one change (create, delete or update records) in an action, you should **batch them**.
```

Comment:
could you just add a line or two of explanation later in this document? It's explained that you can pass prepared operations but nowhere is it explained that you can specifically use `prepareDestroyPermanently` and the other one...